overlord/ifacestate/ifacestate.go

```diff
@@ -150,14 +136,15 @@ func checkConnectConflicts(st *state.State, change *state.Change, plugSnap, slot
 // Connect returns a set of tasks for connecting an interface.
 //
 func Connect(st *state.State, plugSnap, plugName, slotSnap, slotName string) (*state.TaskSet, error) {
-	if err := checkConnectConflicts(st, nil, plugSnap, slotSnap, nil); err != nil {
+	const notAuto = false
```

Comment:
I know what you mean, but the double negative reads a bit awkward.

Would be more natural as "auto = false".